src/com/Jacksonnn/DCCore/Storage/DCPlayerQueries.java

```diff
@@ -0,0 +1,48 @@
+package com.Jacksonnn.DCCore.Storage;
+
+public enum DCPlayerQueries {
```

Comment:
I can't help with SQL with my current understanding, unfortunately

Reply:
rip that’s okay i’m sure you’ll help tons with everything else 🤩